packages/desktop-client/src/components/autocomplete/Autocomplete.tsx

```diff
@@ -41,7 +74,7 @@ function findItem<T extends Item>(
   return value;
 }
 
-function getItemName(item: null | string | Item): string {
+function getItemName<T extends Item>(item: T | T['name']): string {
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏`item` can also be `null`
```suggestion
function getItemName<T extends Item>(item: T | T['name'] | null): string {
```